manage.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-"""django-admin is Django’s command-line utility for administrative tasks."""
+"""Module to manage Django Girls project."""
```

Comment:
I see that it's a module, because it's a `*.py` file. It's pointless to repeat obvious things in docstrings.